signal/playerctl/playerctl_lib.lua

```diff
@@ -59,6 +59,10 @@ fi
 
 tmp_cover_path="${tmp_dir}cover.png"
 
+if [ -f ${tmp_dir}cover.png ]; then
+    rm ${tmp_dir}cover.png
+fi
+
```

Comment:
We shouldn't need this as long as we check for the artUrl being empty before starting a shell to run these commands.